terraform/slurm_cluster/modules/slurm_controller_instance/variables.tf

```diff
@@ -282,6 +282,15 @@ EOD
   default = []
 }
 
+variable "enable_external_prolog_epilog" {
+  description = <<EOD
+Automatically enable a script that will execute prolog and epilog scripts
+shared under /opt/apps from the controller to compute nodes.
```

Comment:
I am a little underwhelmed by the documentation on this feature, especially since the user is taking additional actions to make this work.

To me it kind of reads like I would put a shell script under /opt/apps/<my-script>.sh, but this was not my interpretation from looking at the code. 

Also can we explicitly call out on what machine the scripts run and as what user? I remember seeing a whole matrix of scenarios for prolog/epilog.

If all that is too much for here then you could add a short section in the docs and reference it.

Reply:
I've added a link to the README that will land in #51. I can improve that README by spelling out the directory structure a bit more (while reviewing #51).